packages/run-protocol/test/attestation/test-attestation.js

```diff
@@ -26,23 +24,29 @@ export const makeMockLienBridge = (uBrand, _t) => {
     locked: AmountMath.make(uBrand, 10n),
     unbonding: AmountMath.make(uBrand, 0n),
   };
+  const changeLiened = async (address, delta) => {
+    const currentAmount = liened.get(address) || empty;
+    const current = AmountMath.getValue(uBrand, currentAmount);
+    const changed = current + delta;
+    const changedAmount = AmountMath.make(uBrand, changed);
+    // t.log('changeLiened:', { address, changedAmount });
+    liened.set(address, changedAmount);
+    return changedAmount;
+  };
   return Far('stakeReporter', {
+    increaseLiened: (address, increase) => {
```

Comment:
this and the other mock `stakeReporter` are repeating a lot of the application logic. We've been burned by stale or inaccurate mocks. Would it be feasible to just mock `bridgeManager` and use the real `makeStakeReporter`?  cc @dckc

Reply:
I can see the appeal. It's probably feasible but more than 20 minutes of work. I don't feel strongly about whether it's worthwhile.

Reply:
Yeah, I gave it a try and it's more work than I thought. Withdrawing the suggestion.